File: 백준/Bronze/25372. 성택이의 은밀한 비밀번호/성택이의 은밀한 비밀번호.cc
#include <stdio.h>
#include <string.h>

int main(void) {
	int n;
	char code[21];
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		scanf("%s", code);
		if (strlen(code) >= 6 && strlen(code) <= 9)
			printf("yes\n");
		else
			printf("no\n");
	}
	return 0;
}

File: 백준/Bronze/25372. 성택이의 은밀한 비밀번호/성택이의 은밀한 비밀번호.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        for(int i = 0; i < n; i++) {
            String code = br.readLine();
            if(code.length() >= 6 && code.length() <= 9)
                System.out.println("yes");
            else System.out.println("no");
        }
    }
}